README.md

```diff
@@ -26,22 +26,13 @@ The fields in the table below can be used in these parts of fiboa documents:
 
 | Property Name   | Type   | Description |
 | --------------- | ------ | ----------- |
-| ml:creation_method | string | **REQUIRED**. how the label was created e.g. field walk, image-annotated, ML generated |
-| ml:creation_method_detail | string |  additional details about the creation method, including potentially a link to the algorithm |
 | ml:boundary_type | string | **REQUIRED**. whether the boundary is for a field, ownership parcel, hedge, or other boundary |
 | ml:creation_date | datetime  | **REQUIRED**.[Auto-added] Date collection was created or published |
```

Comment:
The [Auto-added] is confusing. This should probably be removed?

Reply:
We assumed there could possibly be an automated process in using the extension or formatting the data into the fiboa format, but I do think it is better to remove it.

Reply:
Agreed, do you remove it?

Reply:
I did, just haven't committed it yet. Will do so very shortly.